cpg_infra/billing_aggregator/aggregate/aggregate_daily_extended_view.txt

```diff
@@ -24,7 +24,7 @@ MIN(usage_start_time) as usage_start_time,
 MAX(usage_end_time) as usage_end_time,
 SUM(cost) as cost
 FROM `%AGGREGATE_TABLE%`
-WHERE NOT REGEXP_CONTAINS(LOWER(service.description), r'credit|distributed')
+WHERE NOT REGEXP_CONTAINS(LOWER(service.description), r'credit')
```

Comment:
Do you want to keep this bit?

Reply:
I want to exclude credit only in this view.